.aicoe-ci.yaml

```diff
@@ -0,0 +1,10 @@
+check:
+  - thoth-build
+  - thoth-precommit
+build:
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
```

Comment:
dont we need s2i-minimal-notebook or s2i-lab-elyra image for this?

Reply:
You are right, sorry for confusion, should we create a new one s2i-jupyterlab?

Reply:
actually, the jupyterlab-requirements should be installed in the s2i-minimal-notebook, instead of creating an separate image out of the repository.

Reply:
Sounds good! There might be conflicts regarding dependencies: https://github.com/thoth-station/s2i-minimal-notebook/blob/master/Pipfile, because jupyterlab-requirments works with jupyterlab >= 3 and python 3.8

Reply:
I also noticed pipfile and requirements.in are not in sync in s2i-minimal-notebook repo

Reply:
thanks for notifying will update that :+1:

Reply:
opened issue :) and regarding the constraints, do you want me to open PR with dependencies changes?

Reply:
sure, always feel free to open pr. 
thanks

Reply:
Thanks @harshad16